### PR TITLE
Docs Releases - Search and Migration Guide Updates

### PR DESCRIPTION
**Pull request by Adamj1232**
### Description
Search and Migration Guide Updates
### Checklist
- [x] The version field in `package.json` is incremented following [semantic versioning](https://semver.org/)
- [x] The box that allows repo maintainers to update this PR is checked
- [x] I tested locally to make sure this feature/fix works
- [x] I have run `yarn check` & `yarn build` to confirm there are not any associated errors
- [x] This PR passes the Circle CI checks

### CODE REVIEW

File: docs/src/routes/__layout-homepage.svelte [hunk 1]
```diff
@@ -38,14 +41,10 @@
...
-<SEO
- {title}
- {metadescription}
- {url}
-/>
+<SEO {title} {metadescription} {url} />
 
 <KitDocs>
-  <KitDocsLayout {navbar} --kd-content-max-width={'100%'}>
+  <KitDocsLayout {navbar} --kd-content-max-width={'100%'} search>
```
Comment (gesquinca) [inline docs/src/routes/__layout-homepage.svelte:47]: @Adamj1232 Can we remove this too? It's all related to search, but we don't use it in this layout.

File: docs/src/routes/__layout-homepage.svelte [hunk 2]
```diff
@@ -9,10 +9,12 @@
   import '@svelteness/kit-docs/client/styles/theme.css'
   import '$lib/styles/fonts.css'
   import '$lib/styles/kit-docs.css'
+  import '@docsearch/css'
+  import '@svelteness/kit-docs/client/styles/docsearch.css'
```
Comment (gesquinca) [inline docs/src/routes/__layout-homepage.svelte:13]: @Adamj1232 Can we remove this too? It's all related to search, but we don't use it in this layout.

Comment (Adamj1232) [reply docs/src/routes/__layout-homepage.svelte:13]:
```suggestion
```

File: docs/src/routes/__layout-homepage.svelte (hunk 1 above)
Comment (Adamj1232) [reply docs/src/routes/__layout-homepage.svelte:45]:
```suggestion
  <KitDocsLayout {navbar} --kd-content-max-width={'100%'}>
```

**Review by gesquinca — APPROVED** [review]

**Pull request merged**